keyboards/lazydesigners/bolt/info.json

```
@@ -0,0 +1,59 @@
+{
+    "keyboard_name": "Bolt",
+    "url": "http://lazydesigners.cn",
+    "maintainer": "LAZYDESIGNERS",
+    "width": 14.25,
+    "height": 4,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"K000 (F0,F1)", "x":0, "y":0},
+                {"label":"K001 (F0,F4)", "x":1, "y":0},
+                {"label":"K002 (F0,F5)", "x":2, "y":0},
+                {"label":"K003 (F0,F6)", "x":3, "y":0},
+                {"label":"K004 (F0,F7)", "x":4, "y":0},
+                {"label":"K005 (F0,C6)", "x":5, "y":0},
+                {"label":"K006 (F0,B3)", "x":7.5, "y":0},
+                {"label":"K007 (F0,B7)", "x":8.5, "y":0},
+                {"label":"K008 (F0,D0)", "x":9.5, "y":0},
+                {"label":"K009 (F0,D3)", "x":10.5, "y":0},
+                {"label":"K010 (F0,D2)", "x":11.5, "y":0},
+                {"label":"K011 (F0,D1)", "x":12.5, "y":0, "w":1.75},
+                {"label":"K100 (C7,F1)", "x":0, "y":1, "w":1.25},
+                {"label":"K101 (C7,F4)", "x":1, "y":1},
+                {"label":"K102 (C7,F5)", "x":2, "y":1},
+                {"label":"K103 (C7,F6)", "x":3, "y":1},
+                {"label":"K104 (C7,F7)", "x":4, "y":1},
+                {"label":"K105 (C7,C6)", "x":5, "y":1},
```

Visual alignment fix here. (You have some keys overlapped.)

```suggestion
                {"label":"K101 (C7,F4)", "x":1.25, "y":1},
                {"label":"K102 (C7,F5)", "x":2.25, "y":1},
                {"label":"K103 (C7,F6)", "x":3.25, "y":1},
                {"label":"K104 (C7,F7)", "x":4.25, "y":1},
                {"label":"K105 (C7,C6)", "x":5.25, "y":1},
```